framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -414,6 +414,11 @@
   // Also checks that annotations can be inserted. For each Java file, clears all annotations and
   // reinserts them, then checks if the original and modified ASTs are equivalent.
   "ajavaChecks",
+
+  // Convert type argument inference crashes into warnings. This is a boolean option that is
+  // set to true by default. So, use "-AconvertTypeArgInferenceCrashToWarning=false" to true
+  // this option off.
```

Comment:
What happens if the option is off?  Does the Checker Framework crash?  Is the exception ignored and not shown to the user?